ui/components/component-library/button-icon/button-icon.stories.js

```diff
@@ -60,7 +60,7 @@ export default {
     href: {
       control: 'string',
```

Comment:
Update to correct control type
```suggestion
      control: 'text',
```